.github/workflows/backend.yml

```diff
@@ -209,18 +209,18 @@ jobs:
     steps:
       - name: Checkout sentry
         uses: actions/checkout@v2
-        if: needs.files-changed.outputs.changed == 'true'
+        if: needs.files-changed.outputs.migration_lockfile == 'true'
 
       - name: Setup sentry env (python ${{ matrix.python-version }})
         uses: ./.github/actions/setup-sentry
-        if: needs.files-changed.outputs.changed == 'true'
+        if: needs.files-changed.outputs.migration_lockfile == 'true'
         id: setup
         with:
           python-version: ${{ matrix.python-version }}
           pip-cache-version: ${{ secrets.PIP_CACHE_VERSION }}
 
       - name: Migration & lockfile checks
-        if: needs.files-changed.outputs.changed == 'true'
+        if: needs.files-changed.outputs.migration_lockfile == 'true'
```

Comment:
This is defined at the top of the file:
https://github.com/getsentry/sentry/blob/9c4b1236acd9102e9c7970c26432b2cb4d948a0a/.github/workflows/backend.yml#L19